tests/app/views/surveys/test_add_survey_submit.py

```diff
@@ -66,3 +91,21 @@ def test_add_survey_submit_fail(self, decrypt_enrolment_code, get_iac_from_enrol
 
         self.assertEqual(response.status_code, 500)
         self.assertTrue('Error 500 - Server error'.encode() in response.data)
+
+    def test_already_enrolled(self, ):
+        with open('tests/test_data/party/business_party.json') as j:
```

Comment:
j should be literally anything else.  Something like `business_party_json` would be a longer name, but you would actually know roughly what you were referring to when looking through the code.
Single letter variables should be very sparsely used (usually for loop counters)